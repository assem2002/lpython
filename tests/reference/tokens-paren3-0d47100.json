{
    "basename": "tokens-paren3-0d47100",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/tokens/errors/paren3.py",
    "infile_hash": "57c6a1bdbc4971271437ae51ef8eef6a26f1061ff38d6be8a281560b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "tokens-paren3-0d47100.stderr",
    "stderr_hash": "cdf04cdb67804d155e043d9966d42bd9046d70d2cd6aecf9201775ea",
    "returncode": 1
}