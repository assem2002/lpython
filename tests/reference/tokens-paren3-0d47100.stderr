tokenizer error: Parenthesis unexpected
 --> tests/tokens/errors/paren3.py:2:1
  |
2 | ] {}}
  | ^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
